packages/office-ui-fabric-react/src/components/Overlay/Overlay.types.ts

```diff
@@ -1,19 +1,68 @@
 import * as React from 'react';
+import { OverlayBase } from './Overlay.base';
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunction } from '../../Utilities';
 
 export interface IOverlay {
 
 }
 
-export interface IOverlayProps extends React.HTMLAttributes<HTMLElement> {
+export interface IOverlayProps extends React.Props<OverlayBase> {
```

Comment:
Was there a reason for this change?

Reply:
I was experimenting with typings and forgot to change it back, though, this is the pattern other components use. Thanks for catching this.